.github/workflows/expected-release-version.yml

```diff
@@ -0,0 +1,102 @@
+# add comment on PR for expected release version based on the commit messages
+
+name: Expected Release Version
+on:
+  - workflow_call
+
+jobs:
+  release-type:
+    name: Check Release Type
+    runs-on: ubuntu-latest
+    outputs:
+      release_level: ${{ steps.release_type.outputs.RELEASE_TYPE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Let's use the latest [checkout@v4](https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4)